homework/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-homeworks-2024",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "description": "React домашние задания на 2025 год",
+  "author": "Dziyana Bortnik",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.21.0",
+    "@types/react": "^19.0.10",
+    "@types/react-dom": "^19.0.4",
+    "@vitejs/plugin-react": "^4.3.4",
+    "eslint": "^9.21.0",
+    "eslint-plugin-react-hooks": "^5.1.0",
+    "eslint-plugin-react-refresh": "^0.4.19",
+    "globals": "^15.15.0",
+    "vite": "^6.2.0"
+  }
+}
```

Comment:
Usually there should be an empty line at the end of file, that comment includes all other files